friends/src/components/Friends/FriendForm.js

```diff
@@ -36,25 +36,35 @@ class FriendForm extends React.Component {
   addFriendToList = e => {
     e.preventDefault();
     this.props.addFriendToList(this.state.friend);
+    this.setState({
+      friend: {
+        name: "",
+        age: "",
+        email: ""
```

Comment:
i think you state of friend will not clear back to empty strings because it only goes one level deep.